### DISCUSSION TITLE
What file icon theme is used in this screenshot in the README?

### DISCUSSION THREAD

**Discussion by unrealapex** [atom/atom#22514] [Q&A] [answered]
I was wondering which file icon theme is used in the screenshot in the README becase I like the theme.
![](https://user-images.githubusercontent.com/378023/49132478-f4b77680-f31f-11e8-9e10-e8454d8d9b7e.png)

**Comment by ThatXliner** [comment, marked as answer]
I'm pretty sure that's the default
